services/management/enclave/src/service.rs

```diff
@@ -90,6 +91,37 @@ impl TeaclaveManagement for TeaclaveManagementService {
         Ok(response)
     }
 
+    // access control: none
+    fn update_input_file(
+        &self,
+        request: Request<UpdateInputFileRequest>,
+    ) -> TeaclaveServiceResponseResult<UpdateInputFileResponse> {
+        let user_id = self.get_request_user_id(request.metadata())?;
+        let request = request.message;
+
+        let old_input_file: TeaclaveInputFile = self
+            .read_from_db(&request.data_id)
+            .map_err(|_| ServiceError::PermissionDenied)?;
+
+        ensure!(
+            old_input_file.owner.contains(&user_id),
```

Comment:
file.owner == vec![user_id]

Reply:
Okay, thank you very much m4sterchain. There is indeed a problem with the update logic, I will modify it.